### DISCUSSION TITLE
Pas de réponses au quizz (2. HTML & CSS)

### DISCUSSION THREAD

**Discussion by Pismice** [web-classroom/web-classroom.github.io#9] [Q&A : Théorie / Cours] [answered]
Bonsoir,
Pour le quizz du chapitre 2 il n y pas les réponses dans les *Speaker Notes*.
Je ne suis pas sur de me réponses je voulais donc savoir si c'était possible d'avoir accès aux réponses officielles ?

![image](https://github.com/web-classroom/web-classroom.github.io/assets/33352709/e43346e5-d55d-4a16-962a-a8fcaca5596a)

Bonne soirée :)

**Comment by OlivierLmr** [comment, marked as answer]
Très bonne remarque, merci. Je les rajouterai dans la journée.
